cmd/kube-apiserver/app/options/features.go

```diff
@@ -0,0 +1,18 @@
+package options
+
+import (
+	genericfeatures "k8s.io/apiserver/pkg/features"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/features"
+)
+
+var featureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
+	features.ExternalTrafficLocalOnly: {Default: true, PreRelease: utilfeature.GA},
```

Comment:
kube-proxy also needs this feature gate. We should verify if other feature gates are used by more than one kube component.